pokemongo_bot/test/socketio-client.py

```diff
@@ -0,0 +1,15 @@
+from socketIO_client import SocketIO, LoggingNamespace, BaseNamespace
+
+
+def on_location(msg):
```

Comment:
Can we please make this a unit test? It should be run as part of travis. Nobody is ever going to go into this file again to run it and make sure it works. An automated test will ensure that nobody ever breaks it.